messages/login/messages.go

```diff
@@ -0,0 +1,18 @@
+package login
+
+const (
+	// general
+	Usage            = "login"
+	ShortDescription = "Authentication to gain access."
+	LongDescription  = "Authentication to access other commands without blocking."
+	Success          = "successfully logged in"
+
+	// flags
+	FlagUsername = "Login user"
```

Comment:
I think we should change this to FlagEmail and the text to:
"Your email address"

Reply:
I prefer to keep the username that is defined in the spreadsheet, if I change it, that's fine with me.

Reply:
Ah. Good catch.
Let's talk tomorrow during the Daily meeting about this :)